test/CSAccounting.t.sol

```diff
@@ -75,6 +75,10 @@ contract CSAccountingInitTest is CSAccountingBaseInitTest {
     function test_initialize_happyPath() public {
         uint256[] memory curve = new uint256[](1);
         curve[0] = 2 ether;
+
+        // cheat to allow implementation initialisation
+        vm.store(address(accounting), INITIALIZABLE_STORAGE, bytes32(0));
```

Comment:
put it to helper func pls

Reply:
maybe more strict way is to create an `InitializableStorage` struct instance and put it to the slot

Reply:
helper created. Let's keep simple bytes32(0)